lib/rmt/cli/sync.rb

```diff
@@ -0,0 +1,20 @@
+class RMT::CLI::Sync < RMT::CLI::Subcommand
+  default_task :scc
+
+  desc 'scc', 'Sync database with SUSE Customer Center', hide: true
+  def scc
+    abort 'This RMT is in offline-mode. Use `sync airgap` if you want to sync from a portable storage.' if Settings.airgap.offline
+    RMT::SCC.new(options).sync
+  end
+
+  desc 'airgap', 'Store or read data at Airgap storage for offline usage'
```

Comment:
`Airgap storage` sounds like it some kind of special storage device. Air gap is a type of network architecture, not a type of a storage device.